OData.Community.ServiceVersioning.V1.md

```diff
@@ -0,0 +1,448 @@
+Given the following version information in metadata:
+
+```xml
+<EntityContainer Name="DefaultContainer">
+
+  <Annotation Term="ServiceVersioning.ServiceVersionInfo">
+    <Record>
+      <PropertyValue Property="CurrentVersion" String="7.2" />
+      <PropertyValue Property="Required" Bool="true" />
+      <PropertyValue Property="VersionQueryStringParameterName" String="api-version" />
+    </Record>
+  </Annotation>
+
+  <Annotation Term="ServiceVersioning.ScopedServiceVersionInfo">
+    <Collection>
+      <Record>
+        <PropertyValue Property="Scope" String="isvsolution1" />
+        <PropertyValue Property="CurrentVersion" String="5.0" />
+        <PropertyValue Property="VersionQueryStringParameterName" String="solution-versions" />
+      </Record>
+      <Record>
+        <PropertyValue Property="Scope" String="isvsolution2" />
+        <PropertyValue Property="CurrentVersion" String="3.1" />
+        <PropertyValue Property="VersionQueryStringParameterName" String="solution-versions" />
+      </Record>
+    </Collection>
+  </Annotation>
+        
+</EntityContainer>
+```
+
+The client should add the solution-versions query string parameter as follows:
+
+```http
+GET /service/Customers?api-version=7.2&solution-versions=isvsolution1%2F5.0%2Cisvsolution2%2F3.1 HTTP/1.1
+Host: odata.org
+OData-Version: 4.0 
+OData-MaxVersion: 4.0 
+Content-Type: application/json
+```
+
+####3.2.9	Example: Client calls a service with service and scope version in shared query string parameter
+Given the following version information in metadata:
+
+```xml
+<EntityContainer Name="DefaultContainer">
+
+  <Annotation Term="ServiceVersioning.ServiceVersionInfo">
+    <Record>
+      <PropertyValue Property="CurrentVersion" String="7.2" />
+      <PropertyValue Property="Required" Bool="true" />
+      <PropertyValue Property="VersionQueryStringParameterName" String="api-version" />
+    </Record>
+  </Annotation>
+
+  <Annotation Term="ServiceVersioning.ScopedServiceVersionInfo">
+    <Collection>
+      <Record>
+        <PropertyValue Property="Scope" String="isvsolution1" />
+        <PropertyValue Property="CurrentVersion" String="5.0" />
+        <PropertyValue Property="VersionQueryStringParameterName" String="api-version" />
+      </Record>
+      <Record>
+        <PropertyValue Property="Scope" String="isvsolution2" />
+        <PropertyValue Property="CurrentVersion" String="3.1" />
+        <PropertyValue Property="VersionQueryStringParameterName" String="api-version" />
+      </Record>
+    </Collection>
+  </Annotation>
+        
+</EntityContainer>
+```
+
+The client should add the solution-versions query string parameter as follows:
+
+```http
+GET /service/Customers?api-version=7.2%2Cisvsolution1%2F5.0%2Cisvsolution2%2F3.1 HTTP/1.1
+Host: odata.org
+OData-Version: 4.0 
+OData-MaxVersion: 4.0 
+Content-Type: application/json
+```
+
+###3.3	Service Responses

```

Comment:
Strictly speaking a service that requires non-standard headers isn't a compliant OData service, so having a "standard way" of making a service non-standard is a contradiction in itself.

I strongly recommend removing the "Required" property.